src/class/ResponseHeader/ResponseHeader.cpp

```diff
@@ -9,7 +9,7 @@ ResponseHeader::ResponseHeader(HTTPStatusCode code, const ErrorPageSet &error_pa
 {
 	_defaultConstructor();
 	setStatusCode(code);
-	if (_isErrorCode(code))
+	if (_isErrorCode(code) || code.get() == HTTPStatusCode::CREATED)
```

Comment:
why code.get() == HTTPStatusCode::CREATED?
it is better to split the code to "error" flow and "good" flow
now it seem they are mixed